include/faabric/util/bytes.h

```diff
@@ -26,18 +26,18 @@ int safeCopyToBuffer(const uint8_t* dataIn,
                      int bufferLen);
 
 template<class T>
-T unalignedRead(const std::byte* bytes)
+T unalignedRead(const uint8_t *bytes)
```

Comment:
As discussed in PR, have switched this back to `uint8_t` for now to fit with rest of the API, but definitely worth moving to `std::byte` wholesale (see https://github.com/faasm/faabric/issues/199).